sdk/purview/purview-catalog-rest/test/public/utils/createClient.browser.ts

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import PurviewCatalog, { PurviewCatalogRestClient } from "../../../src";
+import { ClientSecretCredential } from "@azure/identity";
+import { ClientOptions } from "@azure-rest/core-client";
+import { env } from "@azure-tools/test-recorder";
+import "./env";
+import { createXhrHttpClient } from "@azure/test-utils";
+
+export function createClient(options?: ClientOptions): PurviewCatalogRestClient {
+  const xhrClient = createXhrHttpClient();
+  const credential = new ClientSecretCredential(
+    env.AZURE_TENANT_ID,
+    env.AZURE_CLIENT_ID,
+    env.AZURE_CLIENT_SECRET,
+    { httpClient: xhrClient }
+  );
+  return PurviewCatalog(env.ENDPOINT, credential, {
+    ...options,
+    httpClient: xhrClient,
```

Comment:
@bterlson here's an example of passing in the old client in order to make the test recorder happy, do you like this approach better?

/cc @joheredi @deyaaeldeen

Reply:
I think this is nicer, we are explicit about the intent and expectations.

Reply:
Would you mind adding a comment to remind us to remove this once the tests are migrated to the Proxy tool?

Reply:
We're tracking the migration progress here: https://github.com/Azure/azure-sdk-for-js/issues/19859

Reply:
I like it better. This seems like the way to me.